.github/workflows/ci.yaml

```diff
@@ -36,6 +36,36 @@ jobs:
     - name: cabal-docspec
       run: nix-shell --pure --run cabal-docspec
 
+  cabal-test-941:
```

Comment:
It's not worth testing twice. just rename this job into `cabal-test` and delete the above.